docusaurus/docs/publish-a-plugin/sign-a-plugin.md

```diff
@@ -33,28 +48,30 @@ To verify ownership of your plugin, generate an Access Policy token that you'll
 
 1. [Create a Grafana Cloud account](https://grafana.com/signup).
 
-1. Login into your account and navigate to **My Account > Security > Access Policies**. Click **Create access policy**.
+1. Login into your account, and then go to **My Account > Security > Access Policies**. 
+
+1. Click **Create access policy**.
 
-   Realm: has to be your-org-name (all-stacks)
-   Scope: plugins:write
+   Realm: has to be **your-org-name** (all-stacks)
+   Scope: **plugins:write**
```

Comment:
This currently appears on the
![Uploading Screenshot 2023-10-20 at 14.56.16.png…]()
 same line, maybe needs 2 whitespace characters after all stacks?

Reply:
done.